website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+
+An operator deployed to a heterogeneous cluster may not support all of the node architectures in the cluster. This has scheduling implications for both the operator as well as its operands.
+
```

Comment:
WDYT about something like:


1) All workload images used in the Pods (for Operator and Operands) provide support for multiple platforms. To check what the platforms supported for each image used you can run `docker inspect <image`. 

Example:


2)  All Deployments/Pods (Operator and Operands) MUST have the `nodeAffinity` properly configured. Otherwise, ....

Example: (we can put here in k8s manifest syntax) 

a) Set the node affinity configuration for your Operator/manager by updating the config/manager file, i.e: 

b) Set the node addinity for all Deployment(s)/Pod(s) created by your Operator (Operands):

If your solution is Ansible or Helm language based you will need to ensure that all playbooks or helm charts are properly configuring the node affinity as in the above example. However, if your Operator is Golang based then, you will do that your code source as the following example:

Example for Golang language-based projects:

3) If your solution is integrated and will be distributed via OLM then: 

We need to add here the steps over what they must do to configure the CSV.
But if they update the config manifest and run make bundle all should be properly set.

Reply:
I tried to front-load the coverage for the basic nodeAffinity block, and provide a separate section covering the OLM CSV file.

In the operator sections of each type, I covered each of the changes that would need to be made in each file.

Finally, I added a section near the overview briefly talking about how to looking the architectures supported in a manifest list, as well as the operator-sdk validator for multi-arch. I also provided a forwarding link for the OLM guide.